sqlite_fdw.c

```diff
@@ -5228,7 +5229,11 @@ sqlite_execute_insert(EState *estate,
 #endif
 
 			value = slot_getattr(slots[i], attnum + 1, &isnull);
-			sqlite_bind_sql_var(type, bindnum, value, fmstate->stmt, &isnull);
+#if PG_VERSION_NUM >= 140000
+			sqlite_bind_sql_var(type, pgtypmod, bindnum, value, fmstate->stmt, &isnull, foreignTableId);
+#else
+			sqlite_bind_sql_var(type, pgtypmod, bindnum, value, fmstate->stmt, &isnull, 0);
```

Comment:
It better to use InvalidOid macro instead of magic number 0.

Reply:
Thanks, @nxhai98! Done.

Reply:
Thank for updating.